make/migrations/postgresql/0031_2.0.3_schema.up.sql

```diff
@@ -49,4 +49,9 @@ UPDATE artifact AS art
 UPDATE tag SET
    repository_id=art.repository_id
    FROM artifact as art
-   WHERE tag.artifact_id=art.id AND tag.repository_id!=art.repository_id;
\ No newline at end of file
+   WHERE tag.artifact_id=art.id AND tag.repository_id!=art.repository_id;
+
+/*remove the constraint for name in table 'notification_policy'*/
+ALTER TABLE notification_policy DROP CONSTRAINT notification_policy_name_key;
+/*add union unique constraint for name and project_id in table 'notification_policy'*/
+ALTER TABLE notification_policy ADD UNIQUE(name,project_id);
```

Comment:
we should consider about the migration path: v2.0.0 - v2.0.4 - v2.1.0, these two lines will be executed twice.

The first line is idempotent, but the second line is not. It will generate two indexes like:


```
Indexes:
    “notification_policy_pkey” PRIMARY KEY, btree (id)
    “notification_policy_name_project_id_key” UNIQUE CONSTRAINT, btree (name, project_id)
    “notification_policy_name_project_id_key1” UNIQUE CONSTRAINT, btree (name, project_id)
```